frontend/resourceadm/pages/DeployResourcePage/DeployResourcePage.tsx

```diff
@@ -177,7 +177,7 @@ export const DeployResourcePage = ({
    * @returns a boolean for if it is possible
    */
   const isDeployPossible = (envVersion: string): boolean => {
-    const policyError = validatePolicyData === undefined || validatePolicyData.status === 400;
+    const policyError = validatePolicyData === undefined || validatePolicyData.status !== 200;
```

Comment:
We have a enum in `shared` named `ServerCodes` which I belive can be used here if I understand the code correct? 😄

Reply:
Good idea 😄